colosseum/validators.py

```diff
@@ -103,3 +104,16 @@ def is_color(value):
 
 
 is_color.description = '<color>'
+
+
+def is_shape(value):
+    """Check if given value is a shape and return it."""
+    try:
+        value = parser.rect(value)
```

Comment:
There's a slight inconsistency in language here - is a rect one of multiple shape options? I know `clip-path` allows for other options, but in that context `rect` isn't one of the options. Is there any particular reason to use the word "shape" here? (and also in the exception raised by `parser.rect()`)

Reply:
Fixed!